regexes/device/mobiles.yml

```diff
@@ -1221,6 +1221,72 @@ Dicam:
   device: 'feature phone'
   model: '$1'
 
+# Digma
+Digma:
+  regex: 'Digma[_ ]([^;/]+) Build|HIT HT707[10]MG|CITI 1902 3G|(CITI (?:[A-Z0-9]+) 3G) C[ST]500[67]PG|iDrQ10 3G|iDxD8 3G|iDnD7|HIT 4G HT7074ML|IDX5|(iDx10|iDx7)|MVM900H(?:WZ|C)|MVM908HCZ|iDxD10 3G|(iDxD[45]|iDxQ5)|iDxD7[_ ]3G|PS604M|PT452E|Linx A400 3G LT4001PG|Linx C500 3G LT5001PG|Linx PS474S|NS6902QL|NS9797MG|((?:Optima|Platina|Plane)[ _](?:[EM])?(?:(?:[0-9\.ST]+|Prime))(?:[ _][43]G)?)|(VOX[ _](?:[0-9\.A-Z]+)[_ ][43]G)'
+  device: 'smartphone'
+  models:
+    - regex: 'HIT HT707[10]MG'
+      model: 'HIT 3G'
+      device: 'tablet'
+    - regex: '(IDX5)'
+      model: '$1'
+    - regex: '(iDxD10 3G)'
+      model: '$1'
+      device: 'tablet'
+    - regex: '(VOX[ _](?:[0-9\.A-Z]+)[_ ][43]G)'
+      model: '$1'
+    - regex: 'PS604M'
+      model: 'Linx 6.0'
+    - regex: '(Linx A400 3G) LT4001PG'
+      model: '$1'
+    - regex: '(Linx C500 3G) LT5001PG'
+      model: '$1'
+    - regex: '(Linx PS474S)'
+      model: '$1'
+    - regex: 'PT452E'
+      model: 'Linx 4.5'
+    - regex: 'MVM900H(?:WZ|C)'
+      model: 'iDx8'
+      device: 'tablet'
+    - regex: 'MVM908HCZ'
+      model: 'iDx9'
+      device: 'tablet'
+    - regex: '(iDxD7[_ ]3G)'
+      model: '$1'
+      device: 'tablet'
+    - regex: '(iDx10|iDx7)'
+      model: '$1'
+    - regex: '(iDxD[45]|iDxQ5)'
+      model: '$1'
+    - regex: 'HIT 4G HT7074ML'
+      model: 'HIT 4G'
+      device: 'tablet'
+    - regex: '(CITI 1902 3G)'
+      model: '$1'
+      device: 'tablet'
+    - regex: '(CITI (?:[A-Z0-9]+) 3G) C[ST]500[67]PG'
+      model: '$1'
+      device: 'tablet'
+    - regex: 'iDxD8 3G'
+      model: 'D-Plane2 8'
+      device: 'tablet'
+    - regex: '(iDrQ10 3G|iDnD7)'
+      model: '$1'
+      device: 'tablet'
+    - regex: 'NS6902QL'
+      model: 'Platina 7.2 4G'
+      device: 'tablet'
+    - regex: 'NS9797MG'
+      model: 'Platina 7.2 3G'
+      device: 'tablet'
+    - regex: '((?:Optima|Platina|Plane)[ _](?:[EM])?(?:(?:[0-9\.ST]+|Prime))(?:[ _][43]G)?)'
```

Comment:
can be simplified: `((?:Optima|Platina|Plane)[ _](?:[EM])?(?:[0-9\.ST]+|Prime)(?:[ _][43]G)?)`